test/unit/remote-config/condition-evaluator.spec.ts

```diff
@@ -1127,6 +1128,8 @@ describe('ConditionEvaluator', () => {
           { targets: ['5.0'], actual: 5.0, outcome: true },
           { targets: ['5.12.3'], actual: '5.12.9', outcome: false },
           { targets: ['5.12.3'], actual: '5.12.3.0.0.0.0', outcome: false },
+          { targets: ['5.6.7.8.9'], actual: '5.6.7.8.9', outcome: true },
```

Comment:
Consider adding a test with 5 segments that should not be equal, ie 5.6.7.8.9 and 4.5.6.7.8